app/controllers/katello/api/v2/content_view_filter_rules_controller.rb

```diff
@@ -7,14 +7,13 @@ class Api::V2::ContentViewFilterRulesController < Api::V2::ApiController
     param :content_view_filter_id, :identifier, :desc => N_("filter identifier"), :required => true
     param_group :search, Api::V2::ApiController
     def index
-      ids = ContentViewFilter.rule_ids_for(@filter)
-      results = ids.map { |id| ContentViewFilter.rule_class_for(@filter).find(id) }
-      collection = {
-        :results  => results.uniq,
-        :subtotal => results.count,
-        :total    => results.count
-      }
-      respond :collection => collection
+      respond(collection: scoped_search(index_relation, :name, :asc, resource_class: ContentViewFilter.rule_class_for(@filter)))
+    end
+
+    def index_relation
+      query = ContentViewFilter.rule_class_for(@filter).where(content_view_filter_id: @filter.id)
+      query = query.where(:name => params[:name]) if params[:name]
```

Comment:
is this needed?  It looks like we're relying on search for the name?

Reply:
Not needed, just convenient. If we remove it, I would rather change it to this:

```rb
params[:search] += "name='#{params[:name]'" if params[:name]
```

Basically, we shouldn't support the `name` param if we're going to ignore it.

Reply:
Nah i'm fine with that, then shouldn't we add name to the apipie docs?

Reply:
Ah, you're right. It's *not* listed in the API. Okay, I'll remove this line.